gating/check/run_elk_tests.sh

```diff
@@ -31,6 +31,9 @@ source "$(readlink -f $(dirname ${0}))/../mnaio_vars.sh"
 ## Main --------------------------------------------------------------------
 
 ${MNAIO_SSH} <<EOS
-  cd /opt/rpc-openstack
-  openstack-ansible playbooks/site-logging.yml
+  cd /opt
+  curl https://raw.githubusercontent.com/rcbops/magnanimous-turbo-chainsaw/master/scripts/setup.sh | bash
```

Comment:
We really should not be curling into bash I feel.

Reply:
Talk to @cloudnull :)

Reply:
I'm performing the installation per the way that they are doing it in production.

Reply:
Not changing my vote, @cloudnull is wrong :D

Reply:
In the interest of time I'm going to merge this, as it's they way it's documented and I want to eat our own dog food.  This is a topic that should be discussed with the tools team.

Reply:
@cloudnull @d34dh0r53 please provide a link so I can report this as a bug (security) issue.

Reply:
@prometheanfire https://github.com/rcbops/magnanimous-turbo-chainsaw/blob/master/README.md

Reply:
this isn't in master? are we committing directly to stable branches now?